sql/CMakeLists.txt

```diff
@@ -723,8 +723,11 @@ ELSE()
   SET(MYSQLD_SOURCE main.cc)
 ENDIF()
 
+LIST(APPEND MYSQLD_SOURCE partitioning/partition_base.cc)
+
 MYSQL_ADD_EXECUTABLE(mysqld
   ${MYSQLD_SOURCE} DESTINATION ${INSTALL_SBINDIR} COMPONENT Server)
+TARGET_INCLUDE_DIRECTORIES(mysqld PRIVATE ${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
Because of two reasons: 1) to have the same "#include" block both for 5.7 and 8.0; 2) make partition_base.* "#include" block independent from their position in the source tree.

Reply:
1) 8.0 changed include path conventions and we have to stay consistent within the 8.0 tree 2) can you please elaborate?

Reply:
I have already fixed it.